### PR TITLE
Correct Dependencies List

### PR DESCRIPTION
**Pull request by Maria-Nosyk**

### CODE REVIEW

File: src/get-started/install.md [hunk 1]
```diff
@@ -45,8 +45,8 @@ use
 
 ```shell
 add-apt-repository ppa:exonum/rocksdb
-apt-get install build-essential libsodium-dev libsnappy-dev \
-    librocksdb5.17 pkg-config libprotobuf-dev protobuf-compiler
+apt-get install build-essential libsodium-dev libsnappy-dev libssl-dev \
+    librocksdb5.17 pkg-config
```
Comment (ivan-ochc) [inline src/get-started/install.md]:
```suggestion
    librocksdb5.17 pkg-config clang-7 lldb-7 lld-7
```

Comment (aleksuss) [reply src/get-started/install.md]: @ivan-ochc are you sure that the meta package `build-essential` doesn't contain these packages ?

Comment (ivan-ochc) [reply src/get-started/install.md]: @aleksuss, yes, `build-essential` doesn't contain these packages.

Comment (ivan-ochc) [reply src/get-started/install.md]: @Maria-Nosyk, @aleksuss , seems, that we need only  `pkg-config, libssl-dev, clang-7, lldb-7, lld-7` and protobuf related packages.

**Pull request merged**